pkg/operator/reports_test.go

```diff
@@ -92,3 +96,86 @@ func TestGetNextReportPeriod(t *testing.T) {
 		})
 	}
 }
+
+func TestReportRunningCondition(t *testing.T) {
+	const (
+		namespace      = "default"
+		testReportName = "test-report"
+		testQueryName  = "test-query"
+	)
+
+	schedule := &metering.ReportSchedule{
+		Period: metering.ReportPeriodCron,
+		Cron:   &metering.ReportScheduleCron{Expression: "5 4 * * *"},
+	}
+
+	reportStart := &time.Time{}
+	reportEndTmp := reportStart.AddDate(0, 1, 0)
+	reportEnd := &reportEndTmp
+
+	testTable := []struct {
+		name       string
+		report     *metering.Report
+		expectFlag bool
+	}{
+		{"new report returns false", testhelpers.NewReport(testReportName, namespace, testQueryName, reportStart, reportEnd, v1alpha1.ReportStatus{}, nil), false},
+		{"run-once report returns true", testhelpers.NewReport(testReportName, namespace, testQueryName, reportStart, reportEnd, v1alpha1.ReportStatus{
+			Conditions: []metering.ReportCondition{
+				{
+					Type:               metering.ReportRunning,
+					Status:             "Active",
```

Comment:
Use `k8s.io/api/core/v1`'s `ConditionStatus` (https://github.com/operator-framework/operator-metering/blob/master/vendor/k8s.io/api/core/v1/types.go#L2229:6) for the value here.